running locally/docker-prestissimo/docker-compose.yml

```diff
@@ -0,0 +1,119 @@
+    ports:
+    - '9083:9083'
+    depends_on:
+      - minio
+    networks:
+      presto:
+        ipv4_address: 172.19.0.3
+  coordinator:
+    container_name: coordinator
+    image: prestodb/presto:0.282
+    platform: linux/amd64
+    restart: always
+    deploy:
+      resources:
+        limits:
+          memory: 2G
+    ports:
+      - 8080:8080
+    volumes:
+      - ./prestissimo/coordinator/log.properties:/opt/presto-server/etc/log.properties
+      - ./prestissimo/coordinator/config.properties:/opt/presto-server/etc/config.properties
+      - ./prestissimo/coordinator/jvm.config:/opt/presto-server/etc/jvm.config
+      - ./prestissimo/coordinator/node.properties:/opt/presto-server/etc/node.properties
+      - ./prestissimo/catalog/jmx.properties:/opt/presto-server/etc/catalog/jmx.properties
+      - ./prestissimo/catalog/tpcds.properties:/opt/presto-server/etc/catalog/tpcds.properties
+      - ./prestissimo/catalog/tpch.properties:/opt/presto-server/etc/catalog/tpch.properties
+      - ./prestissimo/catalog/minio.properties:/opt/presto-server/etc/catalog/minio.properties
+    depends_on:
+      - minio
+      - hadoop
+    environment:
+      - AWS_ACCESS_KEY=FEEF05540F5D8BE7EB62
+      - AWS_SECRET_ACCESS_KEY=2509758e5a6160ddf8c3014651b64006eec5213a
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://172.19.0.4:8080"]
+      interval: 5m
+      timeout: 10s
+      retries: 5
+      start_period: 60s
+    networks:
+      presto:
+        ipv4_address: 172.19.0.4
+  worker_1:
+    container_name: worker_1
+    image: prestodb/presto-native:0.283-dev
+    platform: linux/amd64
+    restart: always
+    deploy:
+      resources:
+        limits:
+          memory: 2G
+    volumes:
+      - ./prestissimo/coordinator/log.properties:/opt/presto-server/etc/log.properties
+      - ./prestissimo/workers/1/config_velox.properties:/opt/presto-server/etc/config.properties
+      - ./prestissimo/workers/jvm.config:/opt/presto-server/etc/jvm.config
+      - ./prestissimo/workers/1/node_velox.properties:/opt/presto-server/etc/node.properties
+      # - ./prestissimo/catalog/jmx.properties:/opt/presto-server/etc/catalog/jmx.properties
+      # - ./prestissimo/catalog/tpcds.properties:/opt/presto-server/etc/catalog/tpcds.properties
+      - ./prestissimo/catalog/tpch.properties:/opt/presto-server/etc/catalog/tpch.properties
+      - ./prestissimo/catalog/minio.properties:/opt/presto-server/etc/catalog/minio.properties
+    depends_on:
+      - minio
+      - hadoop
+    environment:
+      - AWS_ACCESS_KEY=FEEF05540F5D8BE7EB62
+      - AWS_SECRET_ACCESS_KEY=2509758e5a6160ddf8c3014651b64006eec5213a
+    networks:
+      presto:
+        ipv4_address: 172.19.0.11
+  mc-job:
+    image: 'minio/mc:latest'
+    environment:
+      MC_HOST_presto: "http://minioadmin:minioadmin@172.19.0.2:9000"
+    volumes:
+      - './minio/data:/minio'
+    depends_on:
+      - minio
+      - hadoop
+    entrypoint: |
```

Comment:
entrypoint override?

Reply:
This was from Christian's presto-minio repo: https://github.com/czentgr/presto-minio/blob/main/docker-compose.yml

This mc-job container was just meant to be ran to create 2 buckets and load some data inside them in MinIO storage and then be able to query it from presto-cli. It was meant just for fun so I'll remove this container.